config/prow/cluster/boskos-janitor.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     app: boskos-janitor
   namespace: test-pods
 spec:
-  replicas: 4  # 4 distributed janitor instances for gke
+  replicas: 4  # 4 distributed janitor instances for gke; recommend min 2, 1 for each 30 projects
```

Comment:
this documentation might be better suited for the kubernetes-sigs/boskos project.